lib/Transport.d.ts

```diff
@@ -52,7 +52,7 @@ export interface RequestEvent<TResponse = Record<string, any>, TContext = unknow
   warnings: string[] | null;
   meta: {
     context: TContext;
-    name: string;
+    name: string | Symbol;
```

Comment:
Kibana bans `Symbol` usage https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md which could lead to problems with a facade around the ES client